packages/service-core/src/storage/SyncRulesBucketStorage.ts

```diff
@@ -217,6 +217,11 @@ export interface CompactOptions {
   /** Minimum of 1 */
   moveBatchQueryLimit?: number;
 
+  /**
+   * Minimof of 1, default of 10.
```

Comment:
```suggestion
   * Minimum of 1, default of 10.
```

Seems like a typo :)